modules/navier_stokes/test/tests/finite_element/ins/hdg/ip/lid-driven/tests

```diff
@@ -0,0 +1,94 @@
+[Tests]
+  design = 'NavierStokesStressIPHDGKernel.md'
+  issues = '#30038'
+  [lid-driven]
+    requirement = 'The system shall be able to solve a lid driven cavity using an interior penalty hybridizable discontinuous Galerkin discretization when'
+    [uncondensed]
+      type = Exodiff
+      input = lid-driven.i
+      exodiff = lid-driven_out.e
+      detail = 'the system is uncondensed, and when'
+    []
+    [condensed]
+      type = Exodiff
+      input = sc-lid-driven.i
+      cli_args = "Outputs/file_base=lid-driven_out"
+      exodiff = lid-driven_out.e
+      prereq = 'lid-driven/uncondensed'
+      detail = 'the system is condensed.'
+      max_threads = 1
+    []
+  []
+  [condensed_symmetric]
+    requirement = "The condensed matrix shall be symmetric, although I don't understand why. Where did the advection go?"
```

Comment:
this wont look good in the SQA doco

advection hits the non-condensed dofs right, and should not be symmetric

maybe tolerances?